### ISSUE TITLE
"Sequence contains more than one element" exception

### ISSUE THREAD

**Issue by wtanksleyjr** [rmcrackan/Libation#76]
I just upgraded to the new version, started it, and got the following exception:

```

************** Exception Text **************
System.InvalidOperationException: Sequence contains more than one element
   at System.Linq.ThrowHelper.ThrowMoreThanOneElementException()
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.Execute[TResult](Expression query)
   at Microsoft.EntityFrameworkCore.Query.Internal.EntityQueryProvider.Execute[TResult](Expression expression)
   at System.Linq.Queryable.SingleOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   at DataLayer.BookQueries.GetBook(IQueryable`1 books, String productId) in C:\Dropbox\DinahsFolder\coding\_NET\Visual Studio 2019\Libation\DataLayer\QueryObjects\BookQueries.cs:line 18
   at DataLayer.BookQueries.GetBook_Flat_NoTracking(LibationContext context, String productId) in C:\Dropbox\DinahsFolder\coding\_NET\Visual Studio 2019\Libation\DataLayer\QueryObjects\BookQueries.cs:line 12
   at ApplicationServices.TransitionalFileLocator.Audio_Exists(String productId) in C:\Dropbox\DinahsFolder\coding\_NET\Visual Studio 2019\Libation\ApplicationServices\TransitionalFileLocator.cs:line 34
   at ApplicationServices.LibraryCommands.getAudioFileState(String productId) in C:\Dropbox\DinahsFolder\coding\_NET\Visual Studio 2019\Libation\ApplicationServices\LibraryCommands.cs:line 183
   at ApplicationServices.LibraryCommands.<>c.<GetCounts>b__10_0(LibraryBook lb) in C:\Dropbox\DinahsFolder\coding\_NET\Visual Studio 2019\Libation\ApplicationServices\LibraryCommands.cs:line 193
   at System.Linq.Parallel.SelectQueryOperator`2.SelectQueryOperatorResults.GetElement(Int32 index)
   at System.Linq.Parallel.QueryResults`1.get_Item(Int32 index)
   at System.Linq.Parallel.PartitionedDataSource`1.ListContiguousIndexRangeEnumerator.MoveNext(T& currentElement, Int32& currentKey)
   at System.Linq.Parallel.StopAndGoSpoolingTask`2.SpoolingWork()
   at System.Linq.Parallel.SpoolingTaskBase.Work()
   at System.Linq.Parallel.QueryTask.BaseWork(Object unused)
   at System.Linq.Parallel.QueryTask.RunTaskSynchronously(Object o)
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)


************** Loaded Assemblies **************
System.Private.CoreLib
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Private.CoreLib.dll
----------------------------------------
Libation
    Assembly Version: 5.4.4.3
    Win32 Version: 5.4.4.3
    CodeBase: file:///C:/dev/Libation/Libation.dll
----------------------------------------
System.Runtime
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 5.0.8.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Windows.Forms.dll
----------------------------------------
System.Windows.Forms.Primitives
    Assembly Version: 5.0.8.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Windows.Forms.Primitives.dll
----------------------------------------
FileManager
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/FileManager.dll
----------------------------------------
InternalUtilities
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/InternalUtilities.dll
----------------------------------------
LibationWinForms
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/LibationWinForms.dll
----------------------------------------
System.ComponentModel.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ComponentModel.Primitives.dll
----------------------------------------
System.Runtime.InteropServices
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.InteropServices.dll
----------------------------------------
System.Drawing.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Drawing.Primitives.dll
----------------------------------------
System.Collections.Specialized
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Collections.Specialized.dll
----------------------------------------
System.Drawing.Common
    Assembly Version: 5.0.0.2
    Win32 Version: 5.0.421.11614
    CodeBase: file:///C:/dev/Libation/System.Drawing.Common.dll
----------------------------------------
System.Threading
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Threading.dll
----------------------------------------
System.Diagnostics.TraceSource
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Diagnostics.TraceSource.dll
----------------------------------------
System.Collections
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Collections.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 13.0.0.0
    Win32 Version: 13.0.1.25517
    CodeBase: file:///C:/dev/Libation/Newtonsoft.Json.dll
----------------------------------------
netstandard
    Assembly Version: 2.1.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/netstandard.dll
----------------------------------------
System.Linq.Expressions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Linq.Expressions.dll
----------------------------------------
System.ComponentModel.TypeConverter
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ComponentModel.TypeConverter.dll
----------------------------------------
System.ObjectModel
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ObjectModel.dll
----------------------------------------
System.IO.FileSystem
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.IO.FileSystem.dll
----------------------------------------
Dinah.Core
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/Dinah.Core.dll
----------------------------------------
System.Threading.Thread
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Threading.Thread.dll
----------------------------------------
System.Runtime.Numerics
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.Numerics.dll
----------------------------------------
System.Private.Uri
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Private.Uri.dll
----------------------------------------
System.Linq
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Linq.dll
----------------------------------------
System.Collections.Concurrent
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Collections.Concurrent.dll
----------------------------------------
System.Runtime.Serialization.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.Serialization.Primitives.dll
----------------------------------------
System.Runtime.Serialization.Formatters
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.Serialization.Formatters.dll
----------------------------------------
Microsoft.Extensions.Configuration.Abstractions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Configuration.Abstractions.dll
----------------------------------------
Serilog
    Assembly Version: 2.0.0.0
    Win32 Version: 2.10.0.0
    CodeBase: file:///C:/dev/Libation/Serilog.dll
----------------------------------------
System.Data.Common
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Data.Common.dll
----------------------------------------
System.Xml.ReaderWriter
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Xml.ReaderWriter.dll
----------------------------------------
System.Private.Xml
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Private.Xml.dll
----------------------------------------
System.ComponentModel
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ComponentModel.dll
----------------------------------------
System.Reflection.Emit.ILGeneration
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Reflection.Emit.ILGeneration.dll
----------------------------------------
System.Reflection.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Reflection.Primitives.dll
----------------------------------------
System.Reflection.Emit.Lightweight
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Reflection.Emit.Lightweight.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Private.CoreLib.dll
----------------------------------------
AudibleApi
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/AudibleApi.dll
----------------------------------------
System.Net.Http
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.Http.dll
----------------------------------------
System.Net.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.Primitives.dll
----------------------------------------
System.Memory
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Memory.dll
----------------------------------------
System.Console
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Console.dll
----------------------------------------
Microsoft.Extensions.Configuration
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Configuration.dll
----------------------------------------
Microsoft.Extensions.Configuration.Json
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Configuration.Json.dll
----------------------------------------
Serilog.Settings.Configuration
    Assembly Version: 3.1.0.0
    Win32 Version: 3.1.0.0
    CodeBase: file:///C:/dev/Libation/Serilog.Settings.Configuration.dll
----------------------------------------
Microsoft.Extensions.DependencyModel
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.DependencyModel.dll
----------------------------------------
Microsoft.Extensions.FileProviders.Abstractions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.FileProviders.Abstractions.dll
----------------------------------------
Microsoft.Extensions.Configuration.FileExtensions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Configuration.FileExtensions.dll
----------------------------------------
Microsoft.Extensions.FileProviders.Physical
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.FileProviders.Physical.dll
----------------------------------------
Microsoft.Extensions.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Primitives.dll
----------------------------------------
System.IO.FileSystem.Watcher
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.IO.FileSystem.Watcher.dll
----------------------------------------
Microsoft.Extensions.FileSystemGlobbing
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.FileSystemGlobbing.dll
----------------------------------------
System.Threading.Overlapped
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Threading.Overlapped.dll
----------------------------------------
Microsoft.Win32.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/Microsoft.Win32.Primitives.dll
----------------------------------------
System.Text.Json
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Text.Json.dll
----------------------------------------
System.Text.Encoding.Extensions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Text.Encoding.Extensions.dll
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Numerics.Vectors.dll
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.CompilerServices.Unsafe.dll
----------------------------------------
System.Buffers
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Buffers.dll
----------------------------------------
Serilog.Sinks.Console
    Assembly Version: 4.0.0.0
    Win32 Version: 4.0.0.0
    CodeBase: file:///C:/dev/Libation/Serilog.Sinks.Console.dll
----------------------------------------
Serilog.Sinks.File
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.0.0
    CodeBase: file:///C:/dev/Libation/Serilog.Sinks.File.dll
----------------------------------------
System.Text.RegularExpressions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Text.RegularExpressions.dll
----------------------------------------
System.Diagnostics.Process
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Diagnostics.Process.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.InteropServices.RuntimeInformation.dll
----------------------------------------
System.Diagnostics.StackTrace
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Diagnostics.StackTrace.dll
----------------------------------------
Octokit
    Assembly Version: 0.50.0.0
    Win32 Version: 0.50.0.0
    CodeBase: file:///C:/dev/Libation/Octokit.dll
----------------------------------------
System.Diagnostics.Tracing
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Diagnostics.Tracing.dll
----------------------------------------
System.Diagnostics.DiagnosticSource
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Diagnostics.DiagnosticSource.dll
----------------------------------------
System.Net.Security
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.Security.dll
----------------------------------------
System.Security.Cryptography.X509Certificates
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Cryptography.X509Certificates.dll
----------------------------------------
System.Net.Sockets
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.Sockets.dll
----------------------------------------
System.Net.NameResolution
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.NameResolution.dll
----------------------------------------
System.Threading.ThreadPool
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Threading.ThreadPool.dll
----------------------------------------
System.Security.Cryptography.Primitives
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Cryptography.Primitives.dll
----------------------------------------
System.Collections.NonGeneric
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Collections.NonGeneric.dll
----------------------------------------
System.Security.Principal.Windows
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Principal.Windows.dll
----------------------------------------
System.Security.Claims
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Claims.dll
----------------------------------------
System.Security.Cryptography.Encoding
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Cryptography.Encoding.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.IO.Compression.dll
----------------------------------------
System.ComponentModel.EventBasedAsync
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ComponentModel.EventBasedAsync.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/Accessibility.dll
----------------------------------------
System.Runtime.Extensions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Runtime.Extensions.dll
----------------------------------------
Microsoft.Win32.SystemEvents
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Win32.SystemEvents.dll
----------------------------------------
System.Resources.Extensions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/System.Resources.Extensions.dll
----------------------------------------
System.Drawing
    Assembly Version: 5.0.8.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Drawing.dll
----------------------------------------
Dinah.Core.WindowsDesktop
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/Dinah.Core.WindowsDesktop.dll
----------------------------------------
DataLayer
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/DataLayer.dll
----------------------------------------
Dinah.EntityFrameworkCore
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/Dinah.EntityFrameworkCore.dll
----------------------------------------
Microsoft.EntityFrameworkCore
    Assembly Version: 5.0.5.0
    Win32 Version: 5.0.521.16102
    CodeBase: file:///C:/dev/Libation/Microsoft.EntityFrameworkCore.dll
----------------------------------------
ApplicationServices
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/ApplicationServices.dll
----------------------------------------
Microsoft.Extensions.Logging.Abstractions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Logging.Abstractions.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Relational
    Assembly Version: 5.0.5.0
    Win32 Version: 5.0.521.16102
    CodeBase: file:///C:/dev/Libation/Microsoft.EntityFrameworkCore.Relational.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Sqlite
    Assembly Version: 5.0.5.0
    Win32 Version: 5.0.521.16102
    CodeBase: file:///C:/dev/Libation/Microsoft.EntityFrameworkCore.Sqlite.dll
----------------------------------------
Microsoft.Extensions.DependencyInjection.Abstractions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.DependencyInjection.Abstractions.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Abstractions
    Assembly Version: 5.0.5.0
    Win32 Version: 5.0.521.16102
    CodeBase: file:///C:/dev/Libation/Microsoft.EntityFrameworkCore.Abstractions.dll
----------------------------------------
Microsoft.Extensions.Caching.Abstractions
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Caching.Abstractions.dll
----------------------------------------
Microsoft.Extensions.DependencyInjection
    Assembly Version: 5.0.0.1
    Win32 Version: 5.0.120.57516
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.DependencyInjection.dll
----------------------------------------
System.Transactions.Local
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Transactions.Local.dll
----------------------------------------
Microsoft.Extensions.Logging
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Logging.dll
----------------------------------------
Microsoft.Extensions.Options
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Options.dll
----------------------------------------
Microsoft.Extensions.Caching.Memory
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/Microsoft.Extensions.Caching.Memory.dll
----------------------------------------
System.ComponentModel.Annotations
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.ComponentModel.Annotations.dll
----------------------------------------
System.Net.NetworkInformation
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.NetworkInformation.dll
----------------------------------------
System.Collections.Immutable
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Collections.Immutable.dll
----------------------------------------
Microsoft.Data.Sqlite
    Assembly Version: 5.0.5.0
    Win32 Version: 5.0.521.16102
    CodeBase: file:///C:/dev/Libation/Microsoft.Data.Sqlite.dll
----------------------------------------
SQLitePCLRaw.core
    Assembly Version: 2.0.4.976
    Win32 Version: 2.0.4.976
    CodeBase: file:///C:/dev/Libation/SQLitePCLRaw.core.dll
----------------------------------------
SQLitePCLRaw.batteries_v2
    Assembly Version: 2.0.4.976
    Win32 Version: 2.0.4.976
    CodeBase: file:///C:/dev/Libation/SQLitePCLRaw.batteries_v2.dll
----------------------------------------
SQLitePCLRaw.provider.dynamic_cdecl
    Assembly Version: 2.0.4.976
    Win32 Version: 2.0.4.976
    CodeBase: file:///C:/dev/Libation/SQLitePCLRaw.provider.dynamic_cdecl.dll
----------------------------------------
SQLitePCLRaw.nativelibrary
    Assembly Version: 2.0.4.976
    Win32 Version: 2.0.4.976
    CodeBase: file:///C:/dev/Libation/SQLitePCLRaw.nativelibrary.dll
----------------------------------------
System.Linq.Queryable
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Linq.Queryable.dll
----------------------------------------
LibationSearchEngine
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/LibationSearchEngine.dll
----------------------------------------
LuceneNet303r2
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/dev/Libation/LuceneNet303r2.dll
----------------------------------------
System.Configuration.ConfigurationManager
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/dev/Libation/System.Configuration.ConfigurationManager.dll
----------------------------------------
System.Net.WebClient
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Net.WebClient.dll
----------------------------------------
System.Security.Cryptography.Algorithms
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Security.Cryptography.Algorithms.dll
----------------------------------------
System.Linq.Parallel
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Linq.Parallel.dll
----------------------------------------
System.Reflection.Metadata
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.Reflection.Metadata.dll
----------------------------------------
System.IO.MemoryMappedFiles
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.821.31504
    CodeBase: file:///C:/dev/Libation/System.IO.MemoryMappedFiles.dll
----------------------------------------

************** JIT Debugging **************
```

**Comment by wtanksleyjr** [comment]
Oh, this happens after displaying the app window (all of the menus are clickable), but before the list of files manages to display. I assume this is within the loading-existing-files code.

**Comment by wtanksleyjr** [comment]
There's essentially nothing in the logs:

```

2021-07-29 11:09:17.764 -07:00 [INF] (at LibationLauncher.Program.logStartupState(FileManager.Configuration)) Begin Libation. {"Version":"5.4.4.3","LogLevel_Verbose_Enabled":false,"LogLevel_Debug_Enabled":false,"LogLevel_Information_Enabled":true,"LogLevel_Warning_Enabled":true,"LogLevel_Error_Enabled":true,"LogLevel_Fatal_Enabled":true,"LibationFiles":"C:\\Users\\wtank\\Libation","BooksDirectory":"D:\\Libation-Books\\Books","InProgress":"C:\\Users\\wtank\\AppData\\Local\\Temp\\Libation","DownloadsInProgressDir":"C:\\Users\\wtank\\AppData\\Local\\Temp\\Libation\\DownloadsInProgress","DownloadsInProgressFiles":0,"DecryptInProgressDir":"C:\\Users\\wtank\\AppData\\Local\\Temp\\Libation\\DecryptInProgress","DecryptInProgressFiles":0}
2021-07-29 11:13:29.088 -07:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MainFormX","newValue":"538"}
```

**Comment by wtanksleyjr** [comment]
I'm doing a fresh reinstall, so hope I've recorded enough.

**Comment by rmcrackan** [comment]
Thanks much. I fixed some stuff in that area. Let me know if it happens again

**Comment by rmcrackan** [comment]
Whoops -- hit the button too early

As in, it should be fixed in the latest: 5.4.8
